### PR TITLE
Add entry points for admin console scripts

### PR DESCRIPTION
**Pull request by mozbhearsum**
This seems like it should work, but doesn't. We end up with errors such as:
```
root@7dc02f518db9:/app# upload_product_details 
Traceback (most recent call last):
  File "/usr/local/bin/upload_product_details", line 33, in <module>
    sys.exit(load_entry_point('shipit-api-admin', 'console_scripts', 'upload_product_details')())
  File "/usr/local/bin/upload_product_details", line 22, in importlib_load_entry_point
    for entry_point in distribution(dist_name).entry_points
  File "/usr/local/lib/python3.8/importlib/metadata.py", line 504, in distribution
    return Distribution.from_name(distribution_name)
  File "/usr/local/lib/python3.8/importlib/metadata.py", line 177, in from_name
    raise PackageNotFoundError(name)
importlib.metadata.PackageNotFoundError: shipit-api-admin
```

...for reasons that aren't clear to me.

### CODE REVIEW

**Comment by rail** [comment]
See also https://github.com/mozilla-releng/shipit/issues/60

**Comment by mozbhearsum** [comment]
I'm going to go ahead and replace the `pip install -e` that's breaking this with a plain `pip install`. I'm not certain #355 is going to work, and I want to make sure we have the ability to do this easily ASAP. We can probably consider backing this out once we know whether or not that works, and if we need to add similar hooks for other commands.

**Comment by mozbhearsum** [comment]
@rail provided a detailed way to fix this in #60

File: api/setup.py [hunk 1]
```diff
@@ -45,13 +45,22 @@ def get_requirements(req_file):
...
 
 # In order to separate the public and admin apps, use `APP_TYPE` environment
 # variable as a most portable way to parameterize setup.py
 if os.environ.get("APP_TYPE") == "admin":
     packages.extend(find_namespace_packages("src", include=["shipit_api.admin"]))
     name = "shipit_api-admin"
     req_file = "base.txt"
+    entry_points = {
+        "console_scripts": [
+            "shipit_upload_product_details = shipit_api.admin.cli:upload_product_details",
```
Comment (JohanLorenzo) [inline api/setup.py:58]: Nit: 

```suggestion
            "shipit_upload_product_details = shipit_api.admin.cli:download_product_details",
```

https://github.com/mozilla-releng/shipit/blob/db32cee78544d13847269c45f39ed7281e837b5a/api/src/shipit_api/admin/cli.py#L61-L65

**Review by JohanLorenzo — APPROVED** [review]
Thank you very much! I tested each entry point and managed to get them running (except one, but it's just a small nit)

:shipit:

File: api/setup.py (hunk 1 above)
Comment (mozbhearsum) [reply api/setup.py:58]: Whoops, good catch!

**Pull request merged**